src/Keboola/Console/Command/QueueMassTerminateJobs.php

```diff
@@ -0,0 +1,75 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Keboola\Console\Command;
+
+use Keboola\JobQueueClient\Client as JobQueueClient;
+use Keboola\JobQueueClient\ListJobsOptions;
+use Keboola\StorageApi\Client as StorageClient;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+
+class QueueMassTerminateJobs extends Command
+{
+    const ARGUMENT_STORAGE_TOKEN = 'storage-token';
+    const ARGUMENT_CONNECTION_URL = 'connection-url';
+    const ARGUMENT_JOB_STATUS = 'job-status';
+
+    protected function configure()
+    {
+        $this
+            ->setName('queue:terminate-jobs')
```

Comment:
Mozna `queue:terminate-job-projects`, kdyz to muze killnout jen joby z jednoho projektu?

Reply:
`terminate-project-jobs`

Reply:
LOL :laughing: jasne